docs/users/security/overview.md

```diff
@@ -0,0 +1,34 @@
+<!--
+order: 1
+-->
+
+# Overview
+
+Learn about security on Evmos {synopsis}
+
+The [Security Policy on Evmos](https://github.com/evmos/evmos/security/policy)
+includes a vulnerability disclosure policy and guidelines for
+reporting vulnerabilities in the Evmos platform.
+It explains the process for reporting and disclosing vulnerabilities through
+the use of the Immunefi bug bounty program and
```

Comment:
```suggestion
the use of the [Immunefi Bug Bounty Program](https://immunefi.com/bounty/evmos/) and
```

Reply:
@fedekunze Initially I added the link, but returns a 404. Should I add it anyways?

Reply:
yeah please do. You can disable the markdown link linter